client/examples/p2p/main.rs

```diff
@@ -77,7 +76,7 @@ pub async fn start_listening_command(
     let multiaddress: Multiaddr = multiaddr.parse()?;
 
     // spawn network actor
-    let network = stronghold.spawn_p2p(Rule::AllowAll, NetworkConfig::default()).await;
+    let network = stronghold.spawn_p2p(NetworkConfig::default(), None).await;
     println!("Network actor spawned: {:?}", network);
```

Comment:
Better remove print statements :)

Reply:
Don't we want to have some print statements in examples?